common/views/components/ApiToolbar/ApiToolbar.tsx

```diff
@@ -37,26 +38,78 @@ const includes = [
  * from the catalogue.
  *
  */
-async function createTzitzitLink(imageId: string): Promise<Prop> {
-  const apiUrl = `https://api.wellcomecollection.org/catalogue/v2/images/${imageId}?include=source.contributors`;
-  const image: Image = await fetch(apiUrl).then(res => res.json());
+
+function setTzitzitParams(data: {
+  title: string;
+  sourceLink: string;
+  licence: string;
+  author?: string;
+}): string {
```

Comment:
One of the neat things you can do in TypeScript is unpack objects by name, which allows you to do something like:

```
function setTzitzitParams({ title, sourceLink, licence, author }: {
  title: string;
  sourceLink: string;
  licence: string;
  author?: string;
}): string {
  …
```

and then you can use `title`, `sourceLink` etc in the body of the function, rather than referring to them by `data`.